ext/intl/collator/collator_is_numeric.c

```diff
@@ -110,10 +110,7 @@ static double collator_u_strtod(const UChar *nptr, UChar **endptr) /* {{{ */
  *
  * Ignores `locale' stuff.
  */
-static zend_long collator_u_strtol(nptr, endptr, base)
-	const UChar *nptr;
-	UChar **endptr;
-	register int base;
+static zend_long collator_u_strtol(const UChar *nptr, UChar **endptr, register int base)
```

Comment:
```suggestion
static zend_long collator_u_strtol(const UChar *nptr, UChar **endptr, int base)
```

Reply:
Should I drive-by remove the usage of register in the function variable declarations too, or leave that for a separate commit?

Reply:
Feel free to kill them.